### PR TITLE
Adds Growth plan template

### PR DESCRIPTION
**Pull request by tobie**
This is a first path at creating a growth plan template for Growth stage projects wanting to join Impact stage.

### CODE REVIEW

**Comment by tobie** [comment]
Suggestion from @mhdawson to have the plan at the bottom and link to it from the table.

**Review by mhdawson — APPROVED** [review]
LGTM +1 from me once my suggestion is incorporated

**Comment by tobie** [comment]
From CPC meeting today:

- Move this PR to project-status
- Add process in the readme which includes:
    - Projects define their desired status themselves.
    - Then get CPC approval for the roadmap

**Review by boneskull — APPROVED** [review]
yes good perfect

**Review by eemeli — APPROVED** [review]
I like this, but shouldn't it go in the project-status repo rather than here?

**Review by kborchers — APPROVED** [review]

**Review by dmethvin — APPROVED** [review]

**Review by christian-bromann — APPROVED** [review]

**Review by joesepi — APPROVED** [review]

**Comment by joesepi** [comment]
Related: #658 
Related: #613

**Review by sendilkumarn — APPROVED** [review]

**Review by designMoreWeb — APPROVED** [review]
Looks good to me 👍🏽

**Comment by joesepi** [comment]
I moved the template to the project-status repo: https://github.com/openjs-foundation/project-status/pull/56

Still need:
* Add process in the readme which includes:
  * Projects define their desired status themselves.
  * Projects ask for CPC approval of the roadmap.

File: GROWTH_PLAN_TEMPLATE.md
Comment (christian-bromann) [inline GROWTH_PLAN_TEMPLATE.md:3]: Based on discussions from #650
```suggestion
_Note: the purpose of the growth plan is to provide a roadmap for a project to move from At Large stage to Impact stage._
```

Comment (christian-bromann) [reply GROWTH_PLAN_TEMPLATE.md:3]: `Growth` is mentioned couple of times more in the following paragraphs

File: GROWTH_PLAN_TEMPLATE.md
Comment (christian-bromann) [inline GROWTH_PLAN_TEMPLATE.md:13]: I would suggest to generalise this a bit more. Production usage might work for Node.js but not for other Node.js based tools.

```suggestion
| Project shows substantial adoption in production environments or usage within the ecosystem | {CURRENT_STATUS} | 2 | {PLAN} | {DATE} | <!--✅--> |
```

File: GROWTH_PLAN_TEMPLATE.md
Comment (christian-bromann) [inline GROWTH_PLAN_TEMPLATE.md:24]: This suggestion might better emphasise the importance of ongoing contributions.

```suggestion
| Healthy number of regular committers from at least two organizations. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |
```

**Review by christian-bromann — COMMENTED** [review]
Some suggestions

**Comment by eemeli** [comment]
@christian-bromann You'll probably want to add your suggestions to openjs-foundation/project-status#56, where this PR has been "moved".

@tobie / @joesepi Perhaps this PR should be closed, to avoid more duplication?

**Comment by tobie** [comment]
Closed in favor of openjs-foundation/project-status#56.

**Pull request closed without merging**
